tests/create_guardduty_detector_feature/main.tf

```
@@ -0,0 +1,23 @@
+# Creates a GuardDuty Malware Protection for this account.
+# - Creates a GuardDuty detector for this account.
+# - Creates a GuardDuty detector feature for this account.
+
+module "malware_protection_plan" {
```

Rename, this does not represent the config under test


```suggestion
module "detector_features" {
```

---

updated